CMakeLists.txt

```diff
@@ -32,28 +32,37 @@ project(
    LANGUAGES CXX
 )
 
+option(ENABLE_TESTING "Build Unit Tests" ON)
```

Comment:
Nit: let's add the `AU_` prefix here.  Note that I did this in ericriff/au#1, which you're welcome to merge into this PR according to the instructions in the summary. :slightly_smiling_face: